google/cloud/compute_v1/services/accelerator_types/transports/rest.py

```diff
@@ -219,10 +304,12 @@ def get(
             response.content, ignore_unknown_fields=True
         )
 
-    def list(
+    def _list(
```

Comment:
This should not be a breaking change as the transport layer (rest.py) is for internal use. If we see this sort of change in the client.py then it would be a breaking change.

Reply:
> This should not be a breaking change as the transport layer (rest.py) is for internal use. If we see this sort of change in the client.py then it would be a breaking change.

The transports are technically public interfaces, even if it's unlikely the average user is using them directly.

This change looks okay since they are no-ops from the user's perspective. The old names are available through the properties at the bottom of the class.